package.json

```diff
@@ -45,46 +46,56 @@
     ]
   },
   "dependencies": {
-    "@jupyterlab/application": "^2.0.0",
-    "@jupyterlab/apputils": "^2.0.0",
-    "@jupyterlab/codeeditor": "^2.0.0",
-    "@jupyterlab/coreutils": "^4.0.0",
-    "@jupyterlab/docmanager": "^2.0.0",
-    "@jupyterlab/docregistry": "^2.0.0",
-    "@jupyterlab/filebrowser": "^2.2.4",
-    "@jupyterlab/fileeditor": "^2.0.0",
-    "@jupyterlab/launcher": "^2.2.5",
-    "@jupyterlab/mainmenu": "^2.2.3",
-    "@jupyterlab/services": "^5.0.0",
-    "@jupyterlab/settingregistry": "^2.0.0",
-    "@jupyterlab/statedb": "^2.0.0",
-    "@jupyterlab/ui-components": "^2.2.3",
-    "@lumino/coreutils": "^1.3.1",
-    "@lumino/disposable": "^1.2.0",
-    "@lumino/domutils": "^1.1.3",
-    "@lumino/messaging": "^1.2.3",
-    "@lumino/widgets": "^1.8.0",
+    "@jupyterlab/application": "^3.0.0",
+    "@jupyterlab/apputils": "^3.0.0",
+    "@jupyterlab/codeeditor": "^3.0.0",
+    "@jupyterlab/coreutils": "^5.0.0",
+    "@jupyterlab/docmanager": "^3.0.0",
+    "@jupyterlab/docregistry": "^3.0.0",
+    "@jupyterlab/filebrowser": "^3.0.0",
+    "@jupyterlab/fileeditor": "^3.0.0",
+    "@jupyterlab/launcher": "^3.0.0",
+    "@jupyterlab/mainmenu": "^3.0.0",
+    "@jupyterlab/services": "^6.0.0",
+    "@jupyterlab/settingregistry": "^3.0.0",
+    "@jupyterlab/statedb": "^3.0.0",
+    "@jupyterlab/ui-components": "^3.0.0",
+    "@lumino/coreutils": "^1.5.3",
+    "@lumino/disposable": "^1.4.3",
+    "@lumino/domutils": "^1.2.3",
+    "@lumino/messaging": "^1.4.3",
+    "@lumino/widgets": "^1.16.1",
     "pdfjs-dist": "2.0.943",
```

Comment:
Mmmm... if we bump this to the latest (well, any release after [2.6.347](https://github.com/mozilla/pdf.js/releases/tag/v2.6.347)), we'd get typings...